hibernate-core/src/main/java/org/hibernate/dialect/DB2iDialect.java

```diff
@@ -45,17 +43,17 @@ public DB2iDialect() {
 	}
 
 	public DB2iDialect(DatabaseVersion version) {
-		super();
-		this.version = version;
+		super(version);
 	}
 
-	public DatabaseVersion getIVersion() {
-		return version;
+	@Override
+	public DatabaseVersion getDB2Version() {
+		return DatabaseVersion.make(9, 0);
```

Comment:
I would appreciate if you could move these `DatabaseVersion.make(9, 0)` into a static final constant